lib/preview-web/src/PreviewWeb.tsx

```diff
@@ -64,6 +64,7 @@ export class PreviewWeb<TFramework extends AnyFramework> extends Preview<TFramew
     this.view = new WebView();
     this.urlStore = new UrlStore();
 
+    // TODO: talk to tom about this
```

Comment:
@tmeasday let's chat about this, how to remove.

Reply:
I think we can just delete these lines 👍